cache/cache_bench_tool.cc

```diff
@@ -960,11 +1014,14 @@ int cache_bench_tool(int argc, char** argv) {
     exit(1);
   }
 
+  if (FLAGS_seed == 0) {
+    FLAGS_seed = static_cast<uint32_t>(port::GetProcessID());
```

Comment:
Process id is quite random too, I'm wondering why this helps with reproducibility.

Reply:
`DEFINE_uint32(seed, 0, "Hashing/random seed to use. 0 = choose at random");`

So by default, it's random(-ish), but we print the seed needed to reproduce that run if needed.